{
  "comments": [
    {
      "key": {
        "uuid": "cadf96e1_f93e9d32",
        "filename": "ibm_vpd_app.cpp",
        "patchSetId": 4
      },
      "lineNbr": 896,
      "author": {
        "id": 1000338
      },
      "writtenOn": "2021-04-21T05:05:32Z",
      "side": 1,
      "message": "We need to callout by \"CALLOUT_INVENTORY_PATH\" here - See line 1064",
      "revId": "650bf73911295496cdc494348c2e060e066b2d33",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}